### ISSUE TITLE
OA whit copter automode issue

### ISSUE THREAD

**Issue by roque-canales** [ArduPilot/ardupilot#20371]
When activating oa with copter 4.1 in automode the copter always stop at waypoint.
Also spline waypoint no work
Can be possible to desactivate the stop?
does the oa funtion can be enhanced for spline work again?

**Comment by rmackay9** [comment]
I don't think that path planning (BendyRuler, Dijkstra's) ever worked well with Spline waypoints but please tell me if you think I'm wrong.

We have an existing issue here https://github.com/ArduPilot/ardupilot/issues/12691

**Comment by manavgandhi17** [comment]
@rmackay9 You are correct. Path planning has never worked with Spline waypoints afaik. This issue is exactly what is being described in #20140.
